make_release.py

```diff
@@ -349,10 +355,8 @@ def parse_commit_line(line: str, authors: Dict[str, str]) -> Change:
 
     # Standardize
     message.strip()
-    commit_hash = commit_hash.strip('"')
+    commit_hash = commit_hash.strip('"\\ \r\n')
```

Comment:
Do you have an example for this? What happens if we use `commit_hash.strip().strip('"')`  instead? Especially the backslash looks odd for a commit hash.

Reply:
I uses git version 2.37.2 (msys64)

Reply:
My issue was with \\ I just completed the clean-up

Reply:
Could you please provide some example for this and add a corresponding test then?

`commit_hash.strip()` makes sure to strip all whitespace, thus, I would propose to use

```suggestion
    commit_hash = commit_hash.strip().strip('"\\')
```

here.